lib/ansible/galaxy/collection.py

```diff
@@ -574,6 +574,11 @@ def verify_collections(collections, search_paths, apis, validate_certs, ignore_e
                     for search_path in search_paths:
                         b_search_path = to_bytes(os.path.join(search_path, namespace, name), errors='surrogate_or_strict')
                         if os.path.isdir(b_search_path):
+                            if not os.path.isfile(os.path.join(to_text(b_search_path), 'MANIFEST.json')):
```

Comment:
We should use `errors=surrogate_or_strict` when dealing with file paths:
```suggestion
                            if not os.path.isfile(os.path.join(to_text(b_search_path, errors='surrogate_or_strict'), 'MANIFEST.json')):
```

Reply:
Oh, oops. Thanks!